src/Stratis.Bitcoin.IntegrationTests.Common/Runners/StratisBitcoinPosRunner.cs

```diff
@@ -19,6 +20,12 @@ public StratisBitcoinPosRunner(string dataDir)
             this.Network = KnownNetworks.StratisRegTest;
```

Comment:
Can `public StratisBitcoinPosRunner(string dataDir)` also be removed?

Reply:
+1